### PR TITLE
Adding dependency on xPSDesiredStateConfiguration (Fixes #10, fixes #11)

### LINKED ISSUE #10: Missing dependency
This module requires 'xPSDesiredStateConfiguration' module to work.

### LINKED ISSUE #11: PowerShell Gallery module missing DSC resources?
If I install the xChrome module into an Azure Automation DSC instance (from the gallery) and attempt to use it to install Chrome as described [here](https://github.com/PowerShell/xChrome#install-the-chrome-browser) Azure DSC automation fails to compile the configuration because it can't find MSFT_xChrome.  It looks like there is something wrong with the contents of what is in the PowerShell gallery because it does not list any "DSC Resources" on the summary page in the gallery.

For example:
[xChrome does not list DSC Resources and module does not work](https://www.powershellgallery.com/packages/xChrome/1.0.2.0)
[xFirefox does not list DSC Resources and module does not work](http://www.powershellgallery.com/packages/xFirefox/1.0.0)
[cChoco does list DSC Resources and module does work](https://www.powershellgallery.com/packages/cChoco/2.0.5.22)

### PR DESCRIPTION
**Pull request by kwirkykat**
This dependency is required since the module calls Import-DscResource xPSDesiredStateConfiguration.
This fixes both #10 and #11.

---

This change is [Reviewable](https://reviewable.io/reviews/powershell/xchrome/12)

### CODE REVIEW

File: xChrome.psd1 [hunk 1]
```diff
@@ -12,7 +12,7 @@
 # RootModule = ''
 
 # Version number of this module.
-ModuleVersion = '1.1.0.0'
+ModuleVersion = '1.0.2.0'
```
Comment (KarolKaczmarek) [inline xChrome.psd1]: Please increase the second number (1.2.0.0). The third one is used for build number

**Comment by KarolKaczmarek** [comment]
Looks good, but please fix version number

File: xChrome.psd1 (hunk 1 above)
Comment (KarolKaczmarek) [inline xChrome.psd1]: Actually, please revert this change. Version should be increased, but only before it's ready to publish. It will be updated automatically by the publishing scripts

Comment (kwirkykat) [inline xChrome.psd1]: I am not incrementing the version number. I'm just updating it to match what is actually on the PS Gallery.

Comment (KarolKaczmarek) [inline xChrome.psd1]: Yup, we shouldn't be doing it here. Version doesn't match what's on PSGallery cause we've updated version to put the build number on the third place and there was no release since that time. The next version of this module published to PSGallery should be 1.1.0.0 (unless there will be some breaking changes implemented till that time - then we would increase to 2.0.0.0)
